src/containers/lock/containers/LockForm/styles.js

```diff
@@ -17,16 +18,37 @@ export const TermTableBody = styled.tbody``;
 
 export const TermTableRow = styled.tr``;
 
-
 export const TermTableCell = styled.td`
     border-bottom: 1px solid rgba(255, 255, 255, 0.2);
     color: ${theme.colors.primary};
     border-bottom: 1px solid ${theme.colors.opacExtraLighterGrey};
-    padding: 1rem;
+    padding: 0.8rem;
 
     &:first-child {
-        width: 40px;
+        width: 10px;
+        padding-left: 0;
+        ${media.phone`
+            padding-right: .1rem;
+        `};
     }
+
+    ${media.phone`
+        &:nth-child(3) {
```

Comment:
These `nth` selectors rely too much on the current text and are not consistent with the header.

Reply:
yea i just used them to nudge things around a tiny bit on the small mobile screens so the table is kinda readable... i think probably this table will be redesigned as well soon?